velox/common/memory/MappedMemory.h

```diff
@@ -263,6 +265,22 @@ class MappedMemory {
 
   virtual void freeContiguous(ContiguousAllocation& allocation) = 0;
 
+  // Allocates 'size'contiguous bytes and returns the pointer to the
+  // first byte. If 'size' is less than 'maxMallocSize', delegates the
+  // allocation to malloc. If the size is above that and below the
+  // largest size class, allocates one element of the next size
+  // class. If 'size' is greater than the largest size class, calls
+  // allocateContiguous(). Returns nullptr if there is no space. The
+  // amount to allocate is subject to the size limit of
+  // 'this'. Trackers are expected to be called before this.
+  void* FOLLY_NULLABLE allocateBytes(uint64_t size, int32_t maxMallocSize);
```

Comment:
Also I am wondering if calling it minMmapSize be more readable? No strong opinion here.